### PR TITLE
Feature/update to polkadot v0 9 24

### PR DESCRIPTION
**Pull request by MRamanenkau**

### CODE REVIEW

**Review by AndreiNavoichyk — DISMISSED** [review]

**Review by AndreiNavoichyk — APPROVED** [review]

File: runtime/cere-dev/src/lib.rs [hunk 1]
```diff
@@ -126,7 +126,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 30800,
+	spec_version: 30900,
```
Comment (khssnv) [inline runtime/cere-dev/src/lib.rs:129]: Dev updates should increment two lower digits only.

File: runtime/cere/src/lib.rs [hunk 2]
```diff
@@ -125,7 +125,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 30800,
+	spec_version: 30900,
```
Comment (khssnv) [inline runtime/cere/src/lib.rs:128]: Should devnet only runtime update be reflected in mainnet runtime code?

Comment (AndreiNavoichyk) [reply runtime/cere/src/lib.rs:128]: Discussed.

File: runtime/cere-dev/src/lib.rs (hunk 1 above)
Comment (AndreiNavoichyk) [reply runtime/cere-dev/src/lib.rs:129]: Discussed.

**Pull request merged**